locales/zh_TW/json.json

```diff
@@ -4,11 +4,11 @@
     "30 Days": "30天",
     "60 Days": "60天",
     "90 Days": "90天",
-    ":amount Total": ":Amount 總計",
-    ":days day trial": ":Days 天試用",
-    ":resource Details": ":Resource 詳情",
-    ":resource Details: :title": ":Resource 詳細資料：:title",
-    ":trialDays Day Trial": ":TrialDays 天試用",
+    ":amount Total": ":amount 總計",
```

Comment:
There is no need to change these values because the script will automatically return them.

https://github.com/Laravel-Lang/common/discussions/7